### PR TITLE
Driver Leftovers.xml - Windows

### PR DESCRIPTION
**Pull request by Chaython**
A cleanerml xml to delete AMD/Nvidia and other driver installation media.
https://github.com/Chaython/Driver-cleanup-CleanerML

Each option will save users several hundred mb to several gigabytes in space, depending on how many drivers they have installed, and the last time they deleted the dirs.

### CODE REVIEW

File: pending/Driver Leftovers.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<cleaner id="Driver Leftovers" os="windows">
```
Comment (az0) [inline pending/Driver Leftovers.xml:17]: Please make the id lowercase and separated by underscore. This helps when running in command line mode, and it's also a convention.

File: pending/Driver Leftovers.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  <label>Driver Leftovers</label>
+  <description>Deletes staging folders with driver installer data.</description>
+  <option id="NVIDIA">
```
Comment (az0) [inline pending/Driver Leftovers.xml:20]: Please make the id lowercase. It's a convention.

File: pending/Driver Leftovers.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  </option>
+  <option id="AMD">
+    <label>AMD</label>
+    <description>Deletes C:\AMD [used for AMD drivers]</description>
+    <action search="walk.top" command="delete" path="C:\AMD"/>
+  </option>
+  <option id="Temp">
+    <label>Temp</label>
+    <description>Deletes C:\Temp [Used for numerous applications, including RTX Voice]</description>
+    <action search="walk.top" command="delete" path="C:\Temp"/>
```
Comment (az0) [inline pending/Driver Leftovers.xml:33]: As you mentioned, `c:\temp\` is used by many applications. They may not be related at all to "driver leftovers." Could you please make this action more specific to "driver leftovers"?

**Comment by az0** [comment]
Please see my comments

File: pending/Driver Leftovers.xml (hunk 3 above)
Comment (Chaython) [reply pending/Driver Leftovers.xml:33]: The only application I had this year was RTX Voice, regardless it's still temporary, usually an installer staging folder

**Comment by Chaython** [comment]
I believe I made, all requested changes. Sorry for the issues. As per the issue, please see my issue with the "Submitting the cleaner: Submit the cleaner to cleanerml on GitHub as an issue (with a link to a gist) or pull request."

**Pull request merged**